### PR TITLE
Disable Cop RSpec/ExpectChange

### PR DESCRIPTION
**Pull request by brunosedler**

### CODE REVIEW

File: config/ext/rspec.yml [hunk 1]
```diff
@@ -298,7 +298,7 @@ RSpec/ExpectActual:
 
 RSpec/ExpectChange:
   Description: Checks for consistent style of change matcher.
-  Enabled: true
+  Enabled: false
```
Comment (denzelem) [inline config/ext/rspec.yml:301]: You might want to add the reason as comment, so the original discussion is easy to lookup.

```suggestion
  Enabled: false # We decided to disable this cop https://github.com/makandra/makandra-rubocop/issues/29
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,15 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
...
 
+
+## 7.1.0 - 2021-11-29
+
+### Compatible changes
+
+- Disable `RSpec/ExpectChange`
+
+
+
```
Comment (denzelem) [inline CHANGELOG.md:19]: nit: You can check the number of newlines for other entries in the CHANGELOG, I think we only used two here.

**Review by denzelem — APPROVED** [review]
LGTM, thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (brunosedler) [reply CHANGELOG.md:19]: I only checked the last two entries and they both had 3 newlines. But you're right, older ones had only 2 newlines. I'll use 2 here and also remove one line for the last two entries in order to keep the style consistent

**Comment by brunosedler** [comment]
Implements changes discussed in https://github.com/makandra/makandra-rubocop/issues/29

**Pull request merged**